membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -60,17 +60,16 @@ class AttributeController(attributesFromZuora: AttributesFromZuora, commonAction
   }
 
   private def lookup(endpointDescription: String, onSuccessMember: Attributes => Result, onSuccessSupporter: Attributes => Result, onNotFound: Result, sendAttributesIfNotFound: Boolean = false) = {
-    AuthAndBackendViaIdapiAction(ContinueRegardlessOfSignInRecency).async { implicit request: AuthAndBackendRequest[AnyContent] =>
-      if(endpointDescription == "membership" || endpointDescription == "features") {
-        DeprecatedRequestLogger.logDeprecatedRequest(request)
-      }
+    AuthAndBackendViaAuthLibAction.async { implicit request =>
+
+      val userHasValidatedEmail = request.user.flatMap(_.statusFields.userEmailValidated).getOrElse(false)
```

Comment:
out of interest, nowadays is this field only true if the email is valid and we are sure the password is owned by the email owner.  or is it still the case that it's possible to validate the email address without knowing the password?

Reply:
I think it's still possible to validate an email via SSO if the user has no password, but as of https://github.com/guardian/identity/pull/1545 it shouldn't be possible to validate via SSO when we don't know the password is owned by the email owner

Reply:
didn't realise the SSO route existed. the route I was thinking of was validation via email link - certainly in the past you could click a verify email link which would show the log in screen, but would already have marked the email address as valid even if you didn't actually log in.

Reply:
I have checked and that route still exists - I have validated 101682274 just by clicking a link in incognito mode and it's showing me the "enter your email address" screen

Reply:
my team'll discuss this: https://trello.com/c/yjif1Gc0/1584-stop-making-it-possible-to-mark-an-account

thanks for walking me through it